bevm/bevm_client.go

```diff
@@ -91,7 +91,7 @@ type EvmAccount struct {
 func NewEvmAccount(privateKey string) (*EvmAccount, error) {
 	privKey, err := crypto.HexToECDSA(privateKey)
 	if err != nil {
-		return nil, err
+		return nil, errors.New("Error decoding private key: " + err.Error())
```

Comment:
I'm surprised that `golint` doesn't complain about error-message starting with a capital letter. I remember it being quite insistant on that one.

Reply:
I am assuming you are referring to [this](https://github.com/golang/go/wiki/CodeReviewComments#error-strings), thank you I was not aware of this guideline, will fix.